### PR TITLE
Add webCompat feature

### PR DESCRIPTION
**Pull request by joshliebe**
Task/Issue URL: https://app.asana.com/0/1202552961248957/1205083099335524/f

### Description
This PR adds the webCompat content scope script feature.

### Steps to test this PR
- [x] Go to dailystar.co.uk
- [x] Verify that the images next to each story load
- [x] Point at the config (Linked in the task) with the feature disabled
- [x] Reload the app
- [x] Verify that the images show a placeholder

### CODE REVIEW

**Comment by joshliebe** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3358** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/3358" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/android/3358?utm_source=stack-comment).

**Comment by karlenDimla** [comment]
@joshliebe I was trying to check this PR (by testing first) however, the step  `the images next to each story load` always fails as it always show the placeholder

File: web-compat/web-compat-store/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (marcosholgado) [inline web-compat/web-compat-store/src/main/AndroidManifest.xml:1]: You don't need a manifest in this module. You can delete and add the namespace in the build.gradle file

File: web-compat/web-compat-impl/build.gradle [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    implementation Google.android.material
+    implementation AndroidX.constraintLayout
+
+    implementation AndroidX.lifecycle.viewModelKtx
+    implementation AndroidX.lifecycle.runtime.ktx
+    implementation AndroidX.lifecycle.commonJava8
+    implementation AndroidX.lifecycle.liveDataKtx
+
+    // WorkManager
+    implementation AndroidX.work.runtimeKtx
```
Comment (marcosholgado) [inline web-compat/web-compat-impl/build.gradle:57]: Do you need the work manager dependencies?

File: web-compat/web-compat-impl/build.gradle [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    implementation project(path: ':feature-toggles-api')
+    implementation project(path: ':app-build-config-api')
+    implementation project(path: ':web-compat-store')
+    implementation project(path: ':content-scope-scripts-api')
+
+    implementation Kotlin.stdlib.jdk7
+
+    implementation AndroidX.core.ktx
+
+    implementation AndroidX.appCompat
```
Comment (marcosholgado) [inline web-compat/web-compat-impl/build.gradle:47]: Do you need all these dependencies? There's no activities or view models so I take we don't need them.

**Review by marcosholgado — APPROVED** [review]
LGTM! Left a few nits.

**Pull request merged**